NEWS.rst

```diff
@@ -2,6 +2,50 @@
+- The new ``--edit`` option of the ``create`` subcommand launches an editor for entering the contents of the newsfragment. (`#275 <https://github.com/hawkowl/towncrier/issues/275>`_)
+- CPython 3.8 and 3.9 are now part of our automated test matrix and are officially supported. (`#291 <https://github.com/hawkowl/towncrier/issues/291>`_)
+- When searching for the project, first check for an existing importable instance.
+  This helps if the version is only available in the installed version and not the source. (`#297 <https://github.com/hawkowl/towncrier/issues/297>`_)
+- Support building with PEP 517. (`#314 <https://github.com/hawkowl/towncrier/issues/314>`_)
+
+
+Bugfixes
+--------
+
+- Configuration errors found during command line execution now trigger a message to stderr and no longer show a traceback. (`#84 <https://github.com/hawkowl/towncrier/issues/84>`_)
+- A configuration error is triggered when the newsfragment files couldn't be discovered. (`#85 <https://github.com/hawkowl/towncrier/issues/85>`_)
+- Invoking towncrier as `python -m towncrier` works. (`#163 <https://github.com/hawkowl/towncrier/issues/163>`_)
+- ``check`` subcommand defaults to UTF-8 encoding when ``sys.stdout.encoding`` is ``None``.
+  This happens, for example, with Python 2 on GitHub Actions or when the output is piped. (`#175 <https://github.com/hawkowl/towncrier/issues/175>`_)
+- Specifying ``title_format`` disables default top line creation to avoid duplication. (`#180 <https://github.com/hawkowl/towncrier/issues/180>`_)
+
+
+Improved Documentation
+----------------------
+
+- The README now mentions the possibility to name the configuration file
+  ``towncrier.toml`` (in addition to ``pyproject.toml``). (`#172 <https://github.com/hawkowl/towncrier/issues/172>`_)
+- ``start_line`` corrected to ``start_string`` in the readme to match the long standing implementation. (`#277 <https://github.com/hawkowl/towncrier/issues/277>`_)
+
+
+Misc
+----
+
+- `#167 <https://github.com/hawkowl/towncrier/issues/167>`_, `#279 <https://github.com/hawkowl/towncrier/issues/279>`_, `#286 <https://github.com/hawkowl/towncrier/issues/286>`_, `#287 <https://github.com/hawkowl/towncrier/issues/287>`_, `#291 <https://github.com/hawkowl/towncrier/issues/291>`_, `#293 <https://github.com/hawkowl/towncrier/issues/293>`_, `#295 <https://github.com/hawkowl/towncrier/issues/295>`_, `#299 <https://github.com/hawkowl/towncrier/issues/299>`_, `#305 <https://github.com/hawkowl/towncrier/issues/305>`_, `#315 <https://github.com/hawkowl/towncrier/issues/315>`_, `#316 <https://github.com/hawkowl/towncrier/issues/316>`_, `#317 <https://github.com/hawkowl/towncrier/issues/317>`_, `#324 <https://github.com/hawkowl/towncrier/issues/324>`_, `#327 <https://github.com/hawkowl/towncrier/issues/327>`_, `#329 <https://github.com/hawkowl/towncrier/issues/329>`_, `#331 <https://github.com/hawkowl/towncrier/issues/331>`_
```

Comment:
Hmm...  Why did I think that misc was ignored...  bah.

Reply:
https://github.com/twisted/towncrier/blob/4143e0ff55177cca5747d18624d614e593ebe3c3/pyproject.toml#L30-L33

Is this supposed to do what's above?  Or totally not include it in the output?  Guess I'll dive into the tests to try to understand.

Reply:
It seems that misc hasn't been used since [17.4.0](https://github.com/twisted/towncrier/blob/4143e0ff55177cca5747d18624d614e593ebe3c3/NEWS.rst#misc-1) but it does look quite intentionally formatted.  I guess this is good.

![image](https://user-images.githubusercontent.com/543719/111928412-e9a52900-8a89-11eb-9d29-5951a5ad09d8.png)

Reply:
This is how towncrier works by default... and how it's used in Twisted (without the extra links).

It only hides the "content" and not the note number itself :p

I am using a custom template in which `showcontent` is completely ignored.

Maybe besides `showcontent` the default template can have a new `ignore` that can be used to now show that type at all in the release notes.

Reply:
Minor comment. Can we manually remove  the misc section?